scripts/download_mers_structures.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import os
+import sys
+
+repo_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(repo_path)
```

Comment:
We shouldn't need to do this, what are you trying to accomplish with this?

Reply:
Issue #48 is raised and we'll be working on solving this using entry points in the future. Once this PR is merged into main.